ironic/drivers/modules/drac/management.py

```diff
@@ -308,7 +331,143 @@ class DracRedfishManagement(redfish_management.RedfishManagement):
     specific incompatibilities and introduction of vendor value added
     should be implemented by this class.
```

Comment:
Can you remove the 4 lines above since it is no longer true with the addition of the code below?

Reply:
Yes, removed